services/PermissionsService.ts

```diff
@@ -9,13 +9,17 @@ export default class PermissionsService {
   }
 
   public static canSeeEventAttendances(user: UserModel): boolean {
-    return user.isAdmin();
+    return user.isAdmin() || user.isMarketing();
```

Comment:
Did we decide on marketing being allowed to see attendances? Even if they can see feedback, not all attendees may submit feedback, so the list wouldn't be 1:1. I might have missed the discussion, but we agreed on this then it's okay.